project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.13
+sbt.version=0.13.16-M1
```

Comment:
Paul Phillips convinced me that this was a good practice in general but in this case specifically, it gives us the cross-publishing functionality necessary to eventually bring `quasar` to use `sbt` `1.0.0`.